File: app/Models/Pharmacy.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Pharmacy extends Model
{
    protected $table = 'pharmacies';
    protected $primaryKey = 'ID';
    protected $fillable = ['PharmaLicense', 'PharmaName', 'Address', 'PhoneNum', 'status', 'pharmacistName','user_id'];
    
    use HasFactory;

    public function medicines()
    {
        return $this->belongsToMany(Medicine::class, 'medicine_pharmacies', 'PharmacyID', 'MedicineID')
            ->withPivot('Price', 'Quantity');
    }
    public function pharmacyOrders()
    {
        return $this->hasMany(PharmacyOrder::class, 'PharmacyID');
    }
}

File: Flutter Apps/v_pharma/lib/Model/Pharmacy.dart
import 'package:v_pharma/Model/Medicine.dart';

class Pharmacy {
  dynamic id;
  dynamic PharmaLicense;
  dynamic PharmaName;
  dynamic PhoneNum;
  bool? status;
  List<Medicine>? listmedicine;
  dynamic pharmacistName;

  Pharmacy({this.PharmaLicense,this.PharmaName,this.PhoneNum,this.id,this.pharmacistName,this.status,this.listmedicine});
}